Book/book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../item'
+
+class Book < Item
+  def initialize(publish_date, publisher, cover_state, _id = Random.rand(1...1000))
+    super(publish_date)
+    @id = Random.rand(1...1000)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  def can_be_archived?
+    super || cover_state == 'bad'
+  end
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use the instance variable  `@cover_state` in the `can_be_archived?` method instead of the `cover_state` itself for ruby best practices.